time_tracker_api/users/users_namespace.py

```diff
@@ -101,11 +115,11 @@ def post(self, user_id, role_id):
         return AzureConnection().update_role(user_id, role_id, is_grant=False)
 
 
-@ns.route('/<string:user_id>/groups/<string:group_id>/is-member-of')
+@ns.route('/<string:user_id>/is-member-of')
 @ns.param('user_id', 'The user identifier')
-@ns.param('group_id', 'The group name identifier')
 class UserInGroup(Resource):
     @ns.doc('user_in_group')
-    def get(self, user_id, group_id):
+    @ns.expect(data_is_user_in_group)
+    def post(self, user_id):
         """Is User in the Group"""
```

Comment:
Let's change """Is User in the Group""" to """Check if user belongs to group"""

Reply:
Ok, it's done.